client/server/middleware/unsupported-browser.js

```diff
@@ -22,26 +32,25 @@ import analytics from 'calypso/server/lib/analytics';
  * listed in package.json. This list only serves as a way to assist users who are
  * using a browser which is definitely broken. It is not a guarantee that things
  * will work flawlessly on newer versions.
+ * @type {Map<string, (ua: UserAgentDetails) => boolean>}
  */
-const UNSUPPORTED_BROWSERS = [
-	'ie <= 11',
-	'edge <= 79',
-	'firefox <= 73',
-	'chrome <= 79',
-	'safari <= 13', // Note: 13.1 IS supported. Browserslist considers Safari point releases as new versions.
```

Comment:
Despite what's claimed here in the code comment, user agents for all Safari 13.x were previously being treated as unsupported. Since `express-useragent` parses iOS and desktop Safari as just "Safari", I opted to simplify this to check `<= 13` across the board, since that's effectively what we were already doing anyways.